server/views/pages/vary/timeline.test.ts

```diff
@@ -51,4 +51,13 @@ describe('Timeline', () => {
     const $ = cheerio.load(compiledTemplate.render(viewContext))
     expect($('.moj-timeline__date').text()).not.toContain('Last update: ')
   })
+  it('should display correct date description for "vary" routes', () => {
+    viewContext = {
+      isVaryJourney: true,
+      licence: { typeCode: 'AP', licenceExpiryDate: '01/01/2022' },
+    }
+    const $ = cheerio.load(compiledTemplate.render(viewContext))
+    expect($('[data-qa=date]').text()).not.toContain('Release date: ')
```

Comment:
Vary journeys should not show Release date but one of the other date descriptions eg 'Licence end date'